cmd/get_group_members_test.go

```diff
@@ -25,44 +25,53 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestGetGroupMembers(t *testing.T) {
+func TestGetMembers(t *testing.T) {
 	setBasicAuthEnvs()
 	tt := []struct {
-		name            string
-		parentCmdFlagKV map[string]string
-		flagsMap        map[string]string
+		name     string
+		flagsMap map[string]string
+		expect   testResult
 	}{
 		{
-			name: "print in yaml",
+			name: "from project print in yaml",
 			flagsMap: map[string]string{
-				"path": "Group1",
-				"out":  "yaml",
+				"out":          "yaml",
+				"from-project": "Group1/Project1",
+				"query":        "amelia",
 			},
+			expect: pass,
 		},
 		{
 			name: "print in json and use all flags",
 			flagsMap: map[string]string{
-				"path": "Group2",
-				"out":  "json",
+				"out":        "json",
+				"from-group": "Group2",
 			},
+			expect: pass,
 		},
 		{
-			name: "print simple table with no flags",
+			name:   "nothing is passed should fail",
+			expect: fail,
+		},
+		{
+			name: "from group and from project is passed should fail",
 			flagsMap: map[string]string{
-				"path": "Group1",
+				"from-group": "Group2",
+				"from-path":  "Group2",
 			},
+			expect: fail,
 		},
 	}
 
 	for _, tc := range tt {
 		t.Run(tc.name, func(t *testing.T) {
 			execT := execTestCmdFlags{
 				t:        t,
-				cmd:      getGroupMembersCmd,
+				cmd:      getMembersCmd,
 				flagsMap: tc.flagsMap,
 			}
 			stdout, execResult := execT.executeCommand()
-			require.Equal(t, pass, execResult,
+			require.Equal(t, tc.expect, execResult,
```

Comment:
Can you add that  `if tc.expect == fail`, use `require.Contains()` to validate if `stdout` has the expected output error string? At least some part of it.